src/main/kotlin/org/pkl/lsp/documentation/toMarkdown.kt

```diff
@@ -229,6 +229,34 @@ private fun showDocCommentAndModule(node: PklNode?, text: String, context: PklPr
       appendLine()
       appendLine()
       append("in [${module.moduleName}](${module.getLocationUri(forDocs = true)})")
+
+      val docUrl = node?.getDocumentationUrl()
+      if (docUrl != null) {
+        val domain =
+          try {
+            java.net.URI(docUrl).host
+          } catch (e: Exception) {
+            "documentation"
+          }
+
+        val linkText =
+          when (node) {
+            is PklModule -> module.moduleName
+            is PklProperty -> node.name
+            is PklMethod -> node.methodHeader.identifier?.text?.let { "$it()" } ?: "method"
```

Comment:
We already have `PklMethodHeader` here, `PklMethod` is the whole method include the method body

```suggestion
```

Reply:
With this removed link text for me instead of `hasUniquePortNames()` defaulted to a module name `k8s.K8sObject` so PklMethodHeader is not matching, in test case:
```
local result = new K8sObject {}.hasUnique<caret>PortNames()
```